include/benchmark/benchmark.h

```diff
@@ -241,6 +241,14 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #define BENCHMARK_GCC_VERSION (__GNUC__ * 100 + __GNUC_MINOR__)
 #endif
 
+#if defined(COMPILER_GCC) || __has_builtin(__builtin_unreachable)
```

Comment:
How will this work?
This has been moved from `src/internal_macros.h`,
which (at the top of the file) has (*)
```
#ifndef __has_feature
#define __has_feature(x) 0
#endif
#ifndef __has_builtin
#define __has_builtin(x) 0
#endif
```
But it includes *this* header before that:
```
#include "benchmark/benchmark.h"
```
So i think you also need to get (move?) the (*) block here.